tests/queries/skip_list.json

```diff
@@ -392,6 +392,9 @@
         "01475_read_subcolumns_storages",
         "01674_clickhouse_client_query_param_cte",
         "01666_merge_tree_max_query_limit",
+        "01710_projections",
```

Comment:
Why don't you implement parsing for ANTLR grammar?

Reply:
Done